src/app/home/home.page.scss

```diff
@@ -1,4 +1,4 @@
-:host {
+.home-page {
   ion-content {
```

Comment:
a regola questo codice non funziona in quanto si riferisce a tutti gli ion-content allinterno di home-page che è un ion content